README.md

```
@@ -1,11 +1,46 @@
+<table><tr>
+  <td><a href="https://hosted.weblate.org/engage/crm-orbit/"><img alt="Translation status" src="https://hosted.weblate.org/widget/crm-orbit/mobile-app/svg-badge.svg" /></a></td>
+  <td><a href="https://github.com/Masked-Kunsiquat/crm-orbit2/actions/workflows/github-code-scanning/codeql"><img alt="CodeQL" src="https://github.com/Masked-Kunsiquat/crm-orbit2/actions/workflows/github-code-scanning/codeql/badge.svg" /></a></td>
+  <td><a href="https://github.com/Masked-Kunsiquat/crm-orbit2/actions/workflows/dependabot/dependabot-updates"><img alt="Dependabot Updates" src="https://github.com/Masked-Kunsiquat/crm-orbit2/actions/workflows/dependabot/dependabot-updates/badge.svg" /></a></td>
+  <td><a href="https://github.com/Masked-Kunsiquat/crm-orbit2/actions/workflows/eslint.yml"><img alt="ESLint" src="https://github.com/Masked-Kunsiquat/crm-orbit2/actions/workflows/eslint.yml/badge.svg" /></a></td>
+  <td><a href="https://app.codacy.com/gh/Masked-Kunsiquat/crm-orbit2/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_grade"><img alt="Codacy Grade" src="https://app.codacy.com/project/badge/Grade/55db8c1a95f74f078cafef5ede9701ff" /></a></td>
+  <td><a href="https://app.codacy.com/gh/Masked-Kunsiquat/crm-orbit2/dashboard?utm_source=gh&utm_medium=referral&utm_content=&utm_campaign=Badge_coverage"><img alt="Codacy Coverage" src="https://app.codacy.com/project/badge/Coverage/55db8c1a95f74f078cafef5ede9701ff" /></a></td>
+</tr></table>
+
+## Account Management
+<table><tr>
+  <td><img src="docs/images/README/Screenshot_20260227_195856_CRMOrbit.jpg" width="200"/></td>
```

markdownlint-cli2 (0.21.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)